File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/term/read/TermReadResponseTest.java
package uk.ac.jisc.rdss.tests.term.read;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class TermReadResponseTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/term/read/response_schema.json";
    }

    @Test
    public void validateResponse() throws Exception {
        validateJson("messages/term/read/response.json");
    }
}

File: message-api-schema-validator/tests/term/term_read_response_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class TermReadResponseTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return "messages/term/read/response_schema.json"

    def test_validate_request(self):
        self.validate_json("messages/term/read/response.json")
